src/pages/home/report/ReportActionsView.js

```diff
@@ -107,7 +108,11 @@ class ReportActionsView extends React.Component {
     componentDidMount() {
         AppState.addEventListener('change', this.onVisibilityChange);
         subscribeToReportTypingEvents(this.props.reportID);
-        this.keyboardEvent = Keyboard.addListener('keyboardDidShow', this.scrollToListBottom);
+        this.keyboardEvent = Keyboard.addListener('keyboardDidShow', () => {
+            if (ReportActionComposeFocusManager.isFocused()) {
+                this.scrollToListBottom();
+            }
```

Comment:
This logic will never run since `ReportActionComposeFocusManager.isFocused` does not return a value 

```js
function isFocused() {
    if (composerRef.current) {
        composerRef.current.isFocused();
    }
}
```